bpf/lib/bpf_cgroup.h

```diff
@@ -391,4 +391,64 @@ __init_cgrp_tracking_val_heap(struct cgroup *cgrp, cgroup_state state)
 	return heap;
 }
 
+/**
+ * __set_task_cgrpid_tracker() Sets the cgrpid_tracker of a task.
+ * It reads tetragon_conf if not available then exit.
```

Comment:
This comment seems to be outdate, since tetragon_conf is not read but passed as an argument.